nix/sources.json

```diff
@@ -13,42 +14,69 @@
     },
     "deployment-nixpkgs": {
         "branch": "nixos-19.09",
+        "builtin": true,
         "description": "A read-only mirror of NixOS/nixpkgs tracking the released channels. Send issues and PRs to",
         "homepage": "https://github.com/NixOS/nixpkgs",
         "owner": "NixOS",
         "repo": "nixpkgs",
         "rev": "5c52b25283a6cccca443ffb7a358de6fe14b4a81",
         "sha256": "0fhbl6bgabhi1sw1lrs64i0hibmmppy1bh256lq8hxy3a2p1haip",
-        "type": "builtin-tarball",
+        "type": "tarball",
         "url": "https://github.com/NixOS/nixpkgs/archive/5c52b25283a6cccca443ffb7a358de6fe14b4a81.tar.gz",
         "url_template": "https://github.com/<owner>/<repo>/archive/<rev>.tar.gz"
     },
     "easy-purescript-nix": {
         "branch": "master",
         "description": "Easy PureScript (and other tools) with Nix",
         "homepage": "",
-        "owner": "justinwoo",
```

Comment:
Maybe we can retain this by setting `builtin: true` here as well...

Reply:
Done.